src/app/app.component.ts

```diff
@@ -11,7 +11,9 @@ import {
   DatasetState,
   getDatasetState,
 } from './store/dataset';
+import * as Settings from '~/store/settings';
 import { getZipState, AddAction } from './store/products';
+import { DOCUMENT } from '@angular/common';
```

Comment:
I keep two sets of import statements, the first is external imports like angular, the second set is local imports from this project. This last import from `@angular/common` goes in the first group, should actually be line 1 to keep alphabetical order.